www/source/index.html.slim

```diff
@@ -243,7 +243,7 @@ header.bg-gradient.margin-top-offset.short-bg.relative
                 code   its('mode') { should cmp 0644 }
                 code end
                 br
-                code describe myapp.conf do
+                code describe parse_config_file('/etc/myapp.conf') do
```

Comment:
That implies that we can load arbitrary config files. While I agree that myapp.conf may not be the best, I would love to have a resource that fits better. Maybe `nginx` resource?

Reply:
sure, I can do that. the trouble is that `nginx_conf` will result in a lengthy example, so I will use an `apache_conf` example instead.